scripts/base/files/x509/__load__.zeek

```diff
@@ -1,4 +1,4 @@
 @load ./main
 @load ./certificate-event-cache
 
-@load ./log-ocsp
+@load ./log-ocsp
```

Comment:
This one is a mistake, right? The only difference is now that the EOL at the end of the file is missing - which should be there.

Reply:
Fixed.